[Python] list methods. code format and proofread (#2877)
exercism/v3: languages/python/exercises/concept/chaitanas-colossal-coaster/.docs/instructions.md

2. person_name is the name of the person whom we need to kick out.

```python
>>> remove_the_mean_person(queue=["Natasha", "Steve", "Eltran", "Wanda", "Rocket"], person_name="Eltran")
["Natasha", "Steve", "Wanda", "Rocket"]
```

# 5. Namefellows

> You may not have seen 2 unrelated people who look exactly the same, but you've _definitely_ seen unrelated people with the same name exact name! It looks like today there is a lot of that happening. You want to know how many times a particular name has occurred in the queue.

Reviewer: ```suggestion
You may not have seen 2 unrelated people who look exactly the same (_name fellows or doppelgängers_), but you've _definitely_ seen unrelated people with the same exact name! It looks like today there are a lot of them. You want to know how many times a particular name has occurred in the queue.
```

# 5. Namefellows

You may not have seen 2 unrelated people who look exactly the same, but you've _definitely_ seen unrelated people with the same name exact name! It looks like today there is a lot of that happening. You want to know how many times a particular name has occured in the queue.
You may not have seen 2 unrelated people who look exactly the same, but you've _definitely_ seen unrelated people with the same name exact name! It looks like today there is a lot of that happening. You want to know how many times a particular name has occurred in the queue.

> Define the `how_many_dopplegangers()` function that takes 2 parameters `queue, person_name`.

Author: `dopplegangers` should be written `doppelgangers` but I changed it to `namefellows` that I find more appropriate and poetic.

> 2. person_name is the name of the person whom you think have been occurring more than once in the queue.

Reviewer: ```suggestion
2. person_name is the name you think might occur more than once in the queue.
```

> Return the number of occurrences of the name, in `int` format.

Reviewer: ```suggestion
Return the number of occurrences of person_name, in `int` format.
```
